samples/system-test/faceDetection.test.js

```diff
@@ -1,88 +1,88 @@
-/**
```

Comment:
Thinking that these tests should probably be re-enabled?

Reply:
@JustinBeckwith 
The face detection test fails as it is looking for the `const outputFile = path.join(__dirname, `../../vision`, `out.png`);` This seems to cause the other Product Search tests to fail. Do let me know how you would like me to proceed with this.

Reply:
Figured out it is the folder - `vision` folder does not exist in the current structure. Please let me know if you would like me to fix it.